packages/utils/src/svg/build.ts

```diff
@@ -102,9 +102,9 @@ export function iconToSVG(
 				// Horizontal flip
 				transformations.push(
 					'translate(' +
-						(box.width + box.left) +
+						(box.width + box.left).toString() +
 						' ' +
-						(0 - box.top) +
+						(0 - box.top).toString() +
 						')'
```

Comment:
@cyberalien This and others like it would look much better as a template literal.
### ie. Something like
```ts
const {width, left, top } = box
const translateString = `translate(${width + left} ${0 - top})`
```
Didn't do that for this PR to ensure no whitespace issues crept up, but I wouldn't mind refactoring this. Lmk if this is fine for now or if I should I refactor.

Reply:
That's a good idea. Support for ancient browsers is no longer needed.